spec/javascripts/live_search_spec.js

```diff
@@ -491,7 +475,7 @@ describe('liveSearch', function () {
       liveSearch.state = { search: 'state' }
 
       expect($('#order option').length).toBe(2)
-      $('#order option').remove() // Delete all the options
+      $('#order').remove() // Delete all the options
       expect($('#order option').length).toBe(0)
```

Comment:
This `expect()` isn't necessary, surely? 🤔

Reply:
It's probably not but I'm going to leave it as it isn't part of this PR.